_episodes/16-file-formats.md

```diff
@@ -46,6 +46,15 @@ parameterization.
 ~~~
 {: .source}
 
+___Note:___ To follow the example below, you need to download the example input file, *file-formats.bam*. The file is available from [___Note:___ To follow the example below, you need to download the example input file, *file-formats.bam*. The file is available from [https://github.com/common-workflow-language/user_guide/blob/gh-pages/_includes/cwl/file-formats.bam
```

Comment:
Double paste?

Reply:
huh. weird. looks like I really banjaxed this PR! I will revert and have another go...

Reply:
No worries, thank you for your continuing contributions!